### PR TITLE
feat(typescript-web): implement browser and Workers runtimes

### PR DESCRIPTION
**Pull request by sxlijin**
## Summary

- integrate the TypeScript Web bridge with the shared engine and TypeScript runtime APIs
- fill out browser and Cloudflare Workers system operations and connect generated SDKs to the functional runtime
- remove the Web and Workers Vitest ignore annotations once the complete SDK surface is available

## Stack

This PR is based on #4091. The parent provides the independently buildable raw WebAssembly runtime, target-gated CFFI hooks, packaging, and focused browser/workerd tests. This PR layers the shared engine and TypeScript API changes plus full generated SDK parity on top.

## Validation

- cargo nextest run -p sdk_test_typescript_web -E all() --no-fail-fast: 26 passed, 0 skipped
- TypeScript Web SDK matrix passes on Linux, macOS, and Windows
- mise run fmt, mise run stow, and mise run clippy pass

## Summary by CodeRabbit

* **New Features**
  * Added Web runtime sysops for HTTP `_fetch/_send` and synchronous filesystem reads.
  * Expanded the Web bridge with runtime initialization, plus exported handle/media utilities and host-call lifecycle support.
  * Added `getVersion`/`flushEvents` and strengthened call-context lifecycle handling for Web.
* **Bug Fixes**
  * Improved host-value/callable validation, stricter error/panic decoding, and safer cancellation/detach behavior.
  * Enhanced type-context conversions for arrays/maps/instance fields.
* **Tests**
  * Expanded SDK Web test coverage (Web + Workers) and improved fixture harness generation.
* **Chores**
  * Refreshed size-gate baselines and updated Web wasm/type declaration sync tooling.

### CODE REVIEW

File: baml_language/sdks/typescript/bridge_typescript/typescript_src/define_function.ts [hunk 1]
```diff
@@ -245,13 +233,12 @@ export function defineFunction(
...
             const argsProto = encodeCallArgs(built.kwargs, { syncMode: true, callId, typeArgs });
             const callCtxBinding = attachCallContext(built.ctx, callId);
-            let resultBytes: Buffer;
             try {
-                resultBytes = rt.callFunctionSync(bamlFqn, argsProto, null, null);
+                const resultBytes = rt.callFunctionSync(bamlFqn, argsProto, null, null);
+                return decodeCallResult(resultBytes);
             } finally {
                 callCtxBinding.detach();
             }
```
[the remark this answers is not in the document]
Comment (sxlijin) [reply baml_language/sdks/typescript/bridge_typescript/typescript_src/define_function.ts:241]: don't modify the preexisting bridge
